opentelekomcloud/data_source_opentelekomcloud_rds_flavors_v1.go

```diff
@@ -0,0 +1,118 @@
+package opentelekomcloud
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+
```

Comment:
run `goimports` (you can probably setup your editor to do this)

Reply:
it seems goimports cannot delete the blank line. I removed manually.